app/models/order.rb

```diff
@@ -33,6 +33,6 @@ def item_total_value_for_merchant(merchant_id)
   end
 
   def self.sort_by_status
-    order("status = 0 DESC, status = 1 DESC, status = 2 DESC, status = 3 DESC")
```

Comment:
This line was throwing that TLDR error message about deprecation... so I changed it to use a symbol and it works still with no errors. Let me know if I misunderstood the setup here.